src/flownet/ert/FLOW_SIMULATION

```diff
@@ -0,0 +1,9 @@
+EXECUTABLE    flownet_run_flow
```

Comment:
What's up with the spaces in this file?

Reply:
Good question - copied from `semeio`. I'll remove some of them.